overlord/ifacestate/handlers.go

```diff
@@ -346,6 +338,11 @@ func (m *InterfaceManager) doConnect(task *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
+	var autoConnect bool
```

Comment:
I think now that the "connect" task is used much more liberally we should double check that we can get the snapInfos of the involved snaps early, before changing the repo.  Also in case it's an autoconnect we should probably log something but ignore the error and do nothing if one of them is gone away since.

Reply:
rest looks good, but this still seems unadressed

Reply:
Ok, this is addressed now. Thanks!